### ISSUE TITLE
question: How to open my app when tapping the notification?

### ISSUE THREAD

**Issue by ndrake** [DavidBriglio/cordova-plugin-foreground-service#4]
Is there a way to make my application open if the user taps the notification?

**Comment by DavidBriglio** [comment]
Currently this plugin does not have the capability to open the app when clicked. I will be adding this in a future version, thank you for bringing it up!

**Comment by alexhisen** [comment]
Yes, opening app from the notification would be very helpful. It's frustrating to users that it doesn't do that now.

**Comment by alexhisen** [comment]
Note that you can work around with https://github.com/Steffaan/cordova-plugin-local-notifications by replacing the sticky notification with one at the same id that handles clicks:
```
window.cordova.plugins.foregroundService.start('title', 'text', 'icon', 1, notificationId);
setTimeout(() => {
  window.cordova.plugins.notification.local.schedule({
      id: notificationId,
      title: 'title',
      text: 'text',
      channel: 'foreground.service.channel',
      sound: false,
      sticky: true,
      wakeup: false,
      icon: 'res://icon.png',
  });
}, 1000);

**Comment by kshabunov** [comment]
@alexhisen, thanks for the idea! Just a small note: per my experience, in the cordova-plugin-local-notifications configuration
`icon: 'res://icon.png'`
should be
`icon: 'res://icon'`
(without the extension).

**Comment by megadrazz** [comment]
### abrir o aplicativo a partir da notificação

@DavidBriglio 
@alexhisen
@kshabunov 

***add*** 
 Intent intent = new Intent(this, MainActivity.class);
        intent.setFlags(Intent.FLAG_ACTIVITY_TASK_ON_HOME);
        PendingIntent pendingIntent = PendingIntent.getActivity(this,0, intent,PendingIntent.FLAG_UPDATE_CURRENT);

***Notification***
.setContentIntent(pendingIntent)

**Android 8 && Android 9 OK!**

**Comment by mm108** [comment]
> Currently this plugin does not have the capability to open the app when clicked. I will be adding this in a future version, thank you for bringing it up!

Has this feature been added? Because tapping on the notification does not open the app.

**Comment by DavidBriglio** [comment]
This has not yet been implemented.
